guides/common/attributes-base.adoc

```diff
@@ -106,9 +106,10 @@
 :postgresql-conf-dir: {postgresql-data-dir}
 :postgresql-log-dir: {postgresql-data-dir}/log
 :PIV: PIV
-:project-client-name: https://yum.theforeman.org/client/{ProjectVersion}/
-:project-client-RHEL7-url: https://yum.theforeman.org/client/{ProjectVersion}/el7/x86_64/foreman-client-release.rpm
-:project-client-RHEL8-url: https://yum.theforeman.org/client/{ProjectVersion}/el8/x86_64/foreman-client-release.rpm
+:project-client-url: https://yum.theforeman.org/client/{ProjectVersion}
+:project-client-name: {project-client-url}[Foreman Client]
+:project-client-RHEL7-url: {project-client-url}/el7/x86_64/foreman-client-release.rpm
+:project-client-RHEL8-url: {project-client-url}/el8/x86_64/foreman-client-release.rpm
```

Comment:
Upstream we have EL repos so I'd prefer to rename them to `project-client-url-el$version`. That way you can simply grep for `project-client-url` and find all usages without having to resort to some regex.

Reply:
As I say below, I would just do a quick process and annihilate these.